pages/review.vue

```diff
@@ -114,27 +131,41 @@ export default {
         this.$store.commit('items/recipientEmail', email)
       }
     },
+    timerange: {
+      get() {
+        return this.$store.getters['items/timerange']
+      },
+      set(timerange) {
+        this.$store.commit('items/timerange', timerange)
+      }
+    },
     deadline: {
       get() {
         return this.$store.getters['items/deadline']
-      },
-      set(date) {
-        this.$store.commit('items/deadline', date)
       }
+    },
+    formattedDate() {
+      return new Date(this.deadline).toLocaleDateString('de')
     }
   },
   created() {
     setupCalendar({ firstDayOfWeek: 2 })
     this.$store.commit('items/deadline', addDays(startOfDay(new Date()), 3))
   },
   methods: {
+    setTimerange() {
+      const timerangeString = new FormData(this.$refs.timerangeForm).get(
```

Comment:
Why is this necessary?

Reply:
Die werte kommen aus einem Formular mit zwei Radio-Buttons

Reply:
https://vuejs.org/v2/guide/forms.html#Radio